Library/Homebrew/test/gpg_spec.rb

```diff
@@ -13,7 +13,12 @@
         shutup do
           subject.create_test_key(dir)
         end
-        expect(dir/".gnupg/secring.gpg").to exist
+
+        if subject.version == Version.create("2.1")
+          expect(dir/".gnupg/pubring.kbx").to be_file
```

Comment:
Why use the longer form when they are equivalent? Is this just a general style element?